src/auth/googleauth.ts

```diff
@@ -453,15 +486,23 @@ export class GoogleAuth {
    * @param {function=} - Optional callback function
    */
   fromAPIKey(
-      apiKey: string, callback?: (err?: Error|null, client?: JWT) => void) {
+      apiKey: string,
```

Comment:
Same. This is a sync function; it would be good to drop the callback parameter as part of the 1.0 milestone. Don't fix here, open a bug.

Reply:
added to #188